### PR TITLE
Feature/account page recent

### PR DESCRIPTION
**Pull request by Ryden-Graham**
Added new array adapter to recent list on the user page, and made it list completed events from the past week (with comments).

### CODE REVIEW

File: CMPUT_301_Project/app/src/main/java/com/example/cmput_301_project/UserProfilePage.java [hunk 1]
```diff
@@ -47,24 +45,41 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        updateProgress(progress);
 
         // create a list of completed habits and add them to the habit LV
-        ListView habitList = findViewById(R.id.habitList);
+//        ListView habitList = findViewById(R.id.habitList);
 
         // TODO: need to use a list of real recently-completed habits instead of examples here
-        String []habits ={"Habit 1", "Habit 2", "Habit 3", "Habit 4", "Habit 5",
-                "Habit 6" ,"Habit 7", "Habit 8", "Habit 9", "Habit 10"};
+//        String []habits ={"Habit 1", "Habit 2", "Habit 3", "Habit 4", "Habit 5",
```
Comment (ccchoi2) [inline CMPUT_301_Project/app/src/main/java/com/example/cmput_301_project/UserProfilePage.java:68]: Remove all this commented code

File: CMPUT_301_Project/app/src/main/java/com/example/cmput_301_project/ProfileHabitAdapter.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+     * @param holder
+     * @param position
+     */
+    @Override
+    public void onBindViewHolder(ProfileHabitAdapter.ItemVH holder, int position) {
+        HabitEvent habitEvent = habitEventList.get(position);
+        holder.habitNameView.setText(habitEvent.getTitle());
+        holder.commentView.setText(habitEvent.getComment());
+        holder.DateView.setText(habitEvent.getDate());
+        boolean isExpanded=habitEventList.get(position).isExpanded();
```
Comment (ccchoi2) [inline CMPUT_301_Project/app/src/main/java/com/example/cmput_301_project/ProfileHabitAdapter.java:53]: Formatting on equals is weird here

**Review by adiep2 — APPROVED** [review]
Looks good!

**Review by ccchoi2 — CHANGES_REQUESTED** [review]

**Review by ccchoi2 — APPROVED** [review]
LGTM

**Pull request merged**